src/agent0/core/hyperdrive/utilities/run_bots/execute_multi_agent_trades.py

```diff
@@ -255,7 +259,12 @@ async def _async_match_contract_call_to_trade(
             if not trade.maturity_time:
                 raise ValueError("Maturity time was not provided, can't close long position.")
             trade_result = await interface.async_close_long(
-                agent, trade.trade_amount, trade.maturity_time, trade.slippage_tolerance, nonce=nonce
+                agent,
+                trade.trade_amount,
+                trade.maturity_time,
+                trade.slippage_tolerance,
+                nonce=nonce,
```

Comment:
I think we intended for these to be flipped, so the order is `slippage, gas_limit, nonce`.

Reply:
moved nonce to the end and also added keywords in 521ea3e4c9f233cb8e22bdfc9db29042702e9980